### DISCUSSION TITLE
Ability to edit all Miis?

### DISCUSSION THREAD

**Discussion by ghost** [Brionjv/Tomodachi-Life-Save-Editor#127] [Ideas]
I'm not 100% sure if this is a feasible idea, but I was wondering if it could be possible to make a feature that unlocks the ability to edit all the Miis on your island. By this, I mean Miis that have been received from being shared (usually QR codes) that the creator has disabled copying for. It might not be possible, but if it is, I thought it could be a useful feature.

**Comment by Brionjv** [comment]
It will be possible but I need times
